### PR TITLE
[Remove Stream/Optional usage] LPS-175357 modules: layout-page-template-admin-web, layout-page-template-service, layout-page-template-test, layout-portlets-web, layout-reports-web, layout-seo-web-test 

### PR DESCRIPTION
**Pull request by JanisZhang**
Resend, thanks!
Previous: https://github.com/liferay-core-infra/liferay-portal/pull/3430
https://issues.liferay.com/browse/LPS-175357

### CODE REVIEW

**Comment by JanisZhang** [comment]
ci:test:echo

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/apps/layout/layout-seo-web-test/src/testIntegration/java/com/liferay/layout/seo/web/internal/servlet/taglib/test/OpenGraphTopHeadDynamicIncludeTest.java [hunk 1]
```diff
@@ -1014,6 +1014,8 @@ public void testIncludeLocalesLayoutTranslatedLanguagesPortletLayout()
 
 		_layout = _layoutLocalService.updateLayout(_layout);
 
+		Set<Locale> locales = new HashSet<>();
```
Comment (dantewang) [inline modules/apps/layout/layout-seo-web-test/src/testIntegration/java/com/liferay/layout/seo/web/internal/servlet/taglib/test/OpenGraphTopHeadDynamicIncludeTest.java:1017]: @JanisZhang Did Brian ask to put the declaration at almost the beginning of the method?

File: modules/apps/layout/layout-reports-web/src/main/java/com/liferay/layout/reports/web/internal/portlet/action/GetLayoutReportsIssuesMVCResourceCommand.java [hunk 2]
```diff
@@ -166,22 +164,16 @@ private JSONObject _fetchLayoutReportIssuesJSONObject(
...
-			layoutReportsDataProvider.getLayoutReportsIssues(
-				resourceBundle.getLocale(), url);
-
-		Stream<LayoutReportsIssue> stream = layoutReportsIssues.stream();
-
 		return JSONUtil.put(
 			"issues",
 			JSONUtil.putAll(
-				stream.map(
+				TransformUtil.transformToArray(
```
Comment (dantewang) [inline modules/apps/layout/layout-reports-web/src/main/java/com/liferay/layout/reports/web/internal/portlet/action/GetLayoutReportsIssuesMVCResourceCommand.java:170]: @JanisZhang Could you use `JSONUtil.toJSONArray()`?

**Comment by dantewang** [comment]
@JanisZhang Please resend as commented, other places look good to me.

File: modules/apps/layout/layout-seo-web-test/src/testIntegration/java/com/liferay/layout/seo/web/internal/servlet/taglib/test/OpenGraphTopHeadDynamicIncludeTest.java (hunk 1 above)
Comment (JanisZhang) [reply modules/apps/layout/layout-seo-web-test/src/testIntegration/java/com/liferay/layout/seo/web/internal/servlet/taglib/test/OpenGraphTopHeadDynamicIncludeTest.java:1017]: Hi @dantewang, Brian's original commit message is `SF, As used`.  From my understanding, it means I should make the declared variable next to its usage. After revisiting this commit,  I think this SF is not necessary. I will drop this commit and resend it. Thanks!

**Pull request closed without merging**